Apps/WebClient/src/ClientApp/app/services/restUserCommentService.ts

```diff
@@ -18,24 +17,10 @@ export class RestUserCommentService implements IUserCommentService {
     this.isEnabled = config.webClient.modules["Comment"];
   }
 
-  public getComments(): Promise<RequestResult<UserComment[]>> {
+  public getCommentsForEntry(parentEntryId: string): Promise<RequestResult<UserComment[]>> {
     return new Promise((resolve, reject) => {
-      if (!this.isEnabled) {
```

Comment:
we will need to have the isEnabled flag in the retrieving methods. That is because the Comment feature depends on the ODR feature so we want to enable/disable based on configuration